### DISCUSSION TITLE
Add useful presets to the Schema Config to promote standards and interoperability between collections

### DISCUSSION THREAD

**Discussion by maxpatiiuk** [specify/specify7#5587] [Feature Requests]
Suggested by some board member during the last board meeting

Schema config tool can be used to promote standards and interoperability between collections.

I am not yet sure how that may look like, but here are some possibilities:
- Add presents of community recognized field labels (can also serve as a quick start for new institutions)
- Add tips on how to write field labels and usage notes
- Recommend using read-only pick lists to constrain input and in the process increase data quality
- Show heads up if changing catalog number formatter or doing something that clearly goes against community standards (not sure what that may mean in practice though)

**Comment by grantfitzsimmons** [comment]
Good ideas. Should be discussed at some point in the future.

**Comment by grantfitzsimmons** [comment]
We could consider creating default mappings to align common Specify fields with [Darwin Core](DWC.tdwg.org/) concepts.
